### PR TITLE
Sbp 329

### PR DESCRIPTION
**Pull request by rbnks**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -119,12 +140,15 @@
 
     <distributionManagement>
         <repository>
-            <id>sonatype-nexus-staging</id>
```
Comment (sieverssj) [inline pom.xml:122]: These changes shouldn't be necessary.

File: pom.xml [hunk 2]
```diff
@@ -137,19 +161,9 @@
     </licenses>
 
     <scm>
-        <url>git@github.com:sfines/maven-zookeeper-plugin.git</url>
```
Comment (sieverssj) [inline pom.xml:140]: These changes shouldn't be necessary.

File: pom.xml [hunk 3]
```diff
@@ -137,19 +161,9 @@
...
-        <connection>scm:git:git@github.com:sfines/maven-zookeeper-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:sfines/maven-zookeeper-plugin.git</developerConnection>
+        <url>git@github.com:w2ogroup/maven-zookeeper-plugin.git</url>
+        <connection>scm:git:git@github.com:w2ogroup/maven-zookeeper-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:w2ogroup/maven-zookeeper-plugin.git</developerConnection>
     </scm>
 
-    <developers>
```
Comment (sieverssj) [inline pom.xml:145]: These changes shouldn't be necessary.

**Comment by rbnks** [comment]
@sieverssj  does this hurt anything? aka. Can i just leave it as is?

**Comment by sieverssj** [comment]
@rbnks it is my understanding when forking a project that you don't change this type of information.  I don't think it will explicitly hurt anything but I don't think it has to change either.

**Comment by rbnks** [comment]
Ok. i'll change it back.

**Pull request merged**